src/stories/Page.jsx

```diff
@@ -1,51 +1,73 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
+import PropTypes from "prop-types";
 
-import { Header } from './Header';
-import './page.css';
+import { Header } from "./Header";
+import "./page.css";
 
 export const Page = ({ user, onLogin, onLogout, onCreateAccount }) => (
   <article>
-    <Header user={user} onLogin={onLogin} onLogout={onLogout} onCreateAccount={onCreateAccount} />
+    <Header
+      user={user}
+      onLogin={onLogin}
+      onLogout={onLogout}
+      onCreateAccount={onCreateAccount}
+    />
 
     <section>
       <h2>Pages in Storybook</h2>
       <p>
-        We recommend building UIs with a{' '}
-        <a href="https://componentdriven.org" target="_blank" rel="noopener noreferrer">
+        We recommend building UIs with a{" "}
+        <a
+          href="https://componentdriven.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
           <strong>component-driven</strong>
-        </a>{' '}
+        </a>{" "}
         process starting with atomic components and ending with pages.
       </p>
       <p>
-        Render pages with mock data. This makes it easy to build and review page states without
-        needing to navigate to them in your app. Here are some handy patterns for managing page data
-        in Storybook:
+        Render pages with mock data. This makes it easy to build and review page
```

Comment:
And again... ;) if we don't need this, you could delete this.

Reply:
Same here - will delete it later!